### PR TITLE
Ocm docs

### PR DESCRIPTION
**Pull request by aduffeck**
Referencing: #7564 and #7580

This PR adds documentation for the ocm service and fixes a environment variable name for an OCM setting.

### CODE REVIEW

File: docs/services/ocm/create_share_flow.md [hunk 1]
```diff
@@ -0,0 +1,65 @@
+---
+title: Create share flow
```
Comment (mmattel) [inline docs/services/ocm/create_share_flow.md:2]:
```suggestion
title: Create Share Flow
```
Headline Capitalisation...

File: services/ocm/README.md [hunk 2]
```diff
@@ -1,3 +1,64 @@
...
+* ocmd: serves the API for managing federated shares
+
+Internal GRPC APIs:
+* ocmproviderauthorizer: manages the list of trusted providers and verifies requests
+* ocminvitemanager: manages the list and state of invite tokens
+* ocmshareprovider: manages ocm shares on the sharer
+* ocmcore: used for creating federated shares on the receiver side
+* authprovider: authenticates webdav requests using the ocm share tokens
+
+## Trust between instances
```
Comment (mmattel) [inline services/ocm/README.md:16]:
```suggestion
## Trust Between Instances
```

File: services/ocm/README.md [hunk 3]
```diff
@@ -1,3 +1,64 @@
...
+                },
+                "api_version": "0.0.1",
+                "host": "example.com"
+            }
+        ]
+    },
+]
+```
+
+## Invitation workflow
```
Comment (mmattel) [inline services/ocm/README.md:50]:
```suggestion
## Invitation Workflow
```

File: services/ocm/README.md [hunk 4]
```diff
@@ -1,3 +1,64 @@
...
+
+Before sharing a resource with a remote user this user has to be invited by the sharer.
+
+In order to do so a POST request is sent to the `generate-invite` endpoint of the sciencemesh API. The generated token is passed on to the receiver, who will then use the `accept-invite` endpoint to accept the invitation. As a result remote users will be added to the `ocminvitemanager` on both sides. See [invitation flow](invitation_flow) for the according sequence diagram.
+
+The data backend of the `ocminvitemanager` is configurable. The only supported backend currently is `json` which stores the data in a json file on disk.
+
+## Creating shares
```
Comment (mmattel) [inline services/ocm/README.md:58]:
```suggestion
## Creating Shares
```

**Comment by mmattel** [comment]
@aduffeck I am wondering why there are changes in an unrealted file (docs/helpers/extended_vars.yaml). I guess that is unintentionally. In that case, you can drop the change to fix the conflict.

**Comment by aduffeck** [comment]
@mmattel good question, it definitely wasn't an intended change. I rebased against master again, the change is now gone.

**Review by kobergj — APPROVED** [review]

**Pull request merged**